src/util/compareThought.ts

```diff
@@ -131,5 +131,16 @@ export const compareReasonable = makeOrderedComparator<string>([
   (a, b) => compareReasonableText(makeReasonable(a), makeReasonable(b)),
 ])
 
+/** A comparator that sorts in descending order. */
+export const compareReasonableDescending = (a: string, b: string): ComparatorValue => {
+  const comparatorValue = compareReasonable(a, b)
+  return comparatorValue !== 0 ?
+    comparatorValue === -1 ? 1 : -1
+    : 0
```

Comment:
Even better, create a higher order function that allows you to reverse a comparator:

```js
const reverse = comp => (a, b) => comp(b, a)
const compareThoughtDescending = reverse(compareThought)
```

Reply:
Brilliant!